test/fixture/parenthesized-const-assertion.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
I copied this from astexplorer's JSON tab and removed a bunch of location info manually. Let me know if there is a better way to generate this.